src/test/javascript/spec/component/quiz-exercise/quiz-exercise.component.spec.ts

```diff
@@ -281,6 +281,22 @@ describe('QuizExercise Management Component', () => {
         expect(quizExerciseService.delete).toHaveBeenCalledOnce();
     });
 
+    it('should delete multiple quizzes', () => {
+        const headers = new HttpHeaders().append('link', 'link;link');
+        jest.spyOn(quizExerciseService, 'delete').mockReturnValue(
+            of(
+                new HttpResponse({
+                    body: {},
+                    headers,
+                }),
+            ),
+        );
+
+        comp.ngOnInit();
+        com.deleteMultipleExercises([{ id: 1 }, { id: 2 }, { id: 3 }] as QuizExercise[], comp.quizExerciseService);
```

Comment:
```suggestion
        comp.deleteMultipleExercises([{ id: 1 }, { id: 2 }, { id: 3 }] as QuizExercise[], comp.quizExerciseService);
```
This is a typo, right?

Reply:
oh yeah, you are right, thanks for finding it 👍